packages/tools/viewer/src/viewer.ts

```diff
@@ -495,8 +521,8 @@ export class Viewer implements IDisposable {
     private _skybox: Nullable<Mesh> = null;
     private _skyboxBlur: number = 0.3;
     private _skyboxVisible: boolean = true;
-    private _skyboxTexture: Nullable<CubeTexture> = null;
-    private _reflectionTexture: Nullable<CubeTexture> = null;
+    private _skyboxTexture: Nullable<CubeTexture | HDRCubeTexture> = null;

```

Comment:
BaseTexture instead ?

Reply:
I did that at first, but `BaseTexture` doesn't have some of the common API between `CubeTexture` and `HDRCubeTexture` (such as `rotationY`).